lib/convict.js

```diff
@@ -293,6 +293,8 @@ function coerce(k, v, schema, instance) {
         v = moment.duration(parseInt(split[0], 10), split[1]).valueOf();
       }
       break;
+    default:
+        // TODO: Should we throw an exception here?
```

Comment:
Right, I didn't want to modify the behavior with this commit, I just wanted to set up "protections" for future commits.